lib/src/lib/button/components/action-button-menu/lab900-action-button-menu.component.ts

```diff
@@ -22,7 +22,7 @@ export class Lab900ActionButtonMenuComponent {
   }
 
   public getDisabled(action: ActionButton): boolean {
-    return readPropValue(action.disabled, this.data);
+    return readPropValue(typeof action.disabled === 'function' ? action.disabled(this.data) : action.disabled, this.data);
```

Comment:
@lab900-frederico-teixeira This check for function is already done inside readPropValue(). So I think none of these changes are actually needed

Reply:
Xii cool, Ive tested yes it works the same 🙈 , closing the PR